packages/icons/stories/icon.doc.stories.mdx

```diff
@@ -0,0 +1,103 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Icon, AddDocumentIcon } from "@jpmorganchase/uitk-icons";
+import { withFlexGap } from "docs/decorators/withFlexGap";
+import { ComponentAnatomy } from "docs/components/ComponentAnatomy";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+import {
+  CharacteristicUsage,
+  CSSClassTable,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Icons/Icon"
+  component={Icon}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Icon
+
+Icon is an important visual representation of an application, a capability, or some other concept or specific entity with meaning on a interface. Icons help users to recognize and understand a concept or an action regardless of linguistic or cultural boundaries.
+
+If an icon is interactive, use [Button](?path=/docs/documentation-core-button--button-variants) (`CTA`, `Primary` or `Secondary` variant).
+
+<Canvas>
+  <Story id="icons-icon--toolkit-icon" />
+</Canvas>
+
+## Use Icon when...
+
+- You need a visual representation of a concept.
+- An icon doesn't need to be interactive.
+
+## Features
+
+We have a default icon set that starts from 12px for high and medium density (MD), then 14px for low density (LD), and 16px for touch density (TD). (Since our base size of high density (HD)/default is 10px but we don’t allow icons size smaller than 12px, hence why here the HD/default will start from 12px, and its 2X is 20px, 3X is 30px etc.)
+You have more size choices by adding multiplies to 2x, 3x, 4x etc.
+
+<Canvas>
+  <Story id="icons-icon--toolkit-icon-multiple-sizes" />
+</Canvas>
+
+We provide two types of icons: default and solid. Some special icons may be excluded from it like `plus` and `loader`.
+
+<Canvas>
+  <Story id="icons-icon--toolkit-types" />
+</Canvas>
+
+To override the default color of the icon, you can use the `color` css variable.
+
+<Canvas>
+  <AddDocumentIcon
+    style={{ "--uitkIcon-color": "var(--uitk-status-info-foreground)" }}
+  />
+</Canvas>
+
+You can supply custom icons as `children` of the `Icon` component by using an `<svg>`. Whenever you add a custom icon you need to pass an `aria-label` to the Icon.
```

Comment:
I still feel like this needs clarification. What does using an <SVG> mean?

Should this be something more like:

```suggestion
You can supply custom icons as `children` of the `Icon` component. `Icon` has the same API as the native `<svg>` element. Whenever you add a custom icon you need to pass an `aria-label` to the Icon.
```

Reply:
I'll update this, @sion-docs / @joshwooding would you say the next paragraph makes sense here or should i move it to accessibility? (or some content in both)

Reply:
I think it's fine for it to be here. Whether that is achieved by putting it directly here or linking to it and moving it to the accessibility section I don't have a preference

Reply:
I'm happy to go with Josh's judgment.

Reply:
happy to leave it here :)